packages/wallet/backend/src/app.ts

```diff
@@ -386,4 +418,21 @@ export class App {
   async processResources() {
     process.nextTick(() => this.processPendingTransactions())
   }
+
+  ensureGateHubProductionEnv = async (
```

Comment:
The middleware was not added in the `middleware` folder since I needed access to the container.